src/modules/esl-utils/dom/events/misc.ts

```diff
@@ -13,6 +13,10 @@ export const getCompositeTarget = (e: CustomEvent): EventTarget | null => {
   return targets ? targets[0] : e.target;
 };
 
+/** See https://github.com/WICG/EventListenerOptions/blob/gh-pages/explainer.md for details */
+const PASSIVE_EVENTS = ['wheel', 'mousewheel', 'touchstart', 'touchmove'];
+export const isPreferPassive = (event: string): boolean => PASSIVE_EVENTS.includes(event);
```

Comment:
just as a suggestion to name `isPassiveEvent` or something like that

Reply:
Renamed to `isPassiveByDefault` to describe method result more accurate